src/Models/code/Transformers/CodeTransformer.ts

```diff
@@ -0,0 +1,28 @@
+import Code from '../Code';
+import CodeInterface from '../../../Interfaces/code/Code';
+import BaseTransformer from '../../../BaseTransformer';
+import TypeInterface from '../../../Interfaces/code/Type';
+import TypeTransformer from './TypeTransformer';
+
+export default class CodeTransformer extends BaseTransformer {
+    /**
+     * Map an order response
+     *
+     * @param code
+     */
+    mapData(code: CodeInterface): Code {
+        return new Code({
+            name: code.name,
+            createdAt: code.createdAt,
+            updatedAt: code.updatedAt,
+            value: code.value,
+            valueTypeId: code.valueTypeId,
+            // has one
+            type: this.includeType(code),
+        });
+    }
+
+    includeType(code: CodeInterface): TypeInterface | null {
```

Comment:
I think this should be returning a `Type`. Anything post transform will be a class, not just a type.

It may be this mistake was made elsewhere too, and this was simply copied. But if you could update this one I'll let people know